packages/sdk/src/protocols/networkStats.ts

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Middleware } from '.';
+import { Message } from '..';
+import { ExportedPromise } from '../utils/exportedPromise';
+import filterEmpty from '../utils/filterEmpty';
+import validateJsonFieldName from '../utils/validateJsonFieldName';
+
+export interface NetworkStatsFrame {
```

Comment:
>interface [](start = 7, length = 9)

For consistency, prefer `type` over `interface`. Reserve `interface` for types that will be implemented by a class.